daffodil-japi/src/main/scala/org/apache/daffodil/japi/Daffodil.scala

```diff
@@ -953,3 +975,68 @@ class DaffodilXMLReader private[japi] (xmlrdr: SDaffodilXMLReader) extends org.x
    */
   def parse(arr: Array[Byte]): Unit = xmlrdr.parse(arr)
 }
+
+/**
+ * Read in an infoset from any SAX Reader for unparsing
+ */
```

Comment:
I'm not sure I would say "read in". Since this doesn't really do that. The XMLReader handles parsing the XML. This just unparses data base on SAX events from an XMLReader.